hnn_core/network.py

```diff
@@ -219,47 +286,411 @@ def __repr__(self):
                  len(self.pos_dict['L5_basket'])))
         return '<%s | %s>' % (class_name, s)
 
-    def _add_external_feeds(self):
-        """Legacy function, for backward compatibility with original HNN-GUI
+    def add_evoked_drive(self, name, mu, sigma, numspikes, weights_ampa,
+                         weights_nmda, location, space_constant=3.,
+                         dispersion_time=0.1, seedcore=42):
+        """Add an 'evoked' external drive to the network
+
+        Parameters
+        ----------
+        name : str
+            Unique name for drive
+        mu : float
+            Mean of Gaussian event time distribution (across target cells)
+        sigma : float
+            Standard deviation of event time distribution (across target cells)
+            Set to zero for "synchronous" evoked drive.
+        numspikes : int
+            Number of spikes at each target cell
+        weights_ampa : dict
+            Synaptic weights of AMPA receptors on each targeted cell type
+            (keys). Weights can be zero or left omitted.
+        weights_nmda : dict
+            Synaptic weights of NMDA receptors on each targeted cell type
+            (keys). Weights can be zero or left omitted.
+        location : str
+            Target location of synapses ('distal' or 'proximal')
+        space_constant : float
+            Describes lateral dispersion (from column origin) of synaptic
+            delays within the simulated column
+        dispersion_time : float or dict
+            Synaptic delay at the column origin, dispersed laterally as a
+            function of the space_constant. If float, applies to all target
+            cell types. Use dict, to create dispersion->cell mapping.
```

Comment:
should we bake this into a single parameter? 

`synaptic_delay=dict(amplitude, space_constant)`

then we have a one-to-one mapping to the math and what actually happens

Reply:
+1 from me, let's see if you find something else though...